smtp.go

```diff
@@ -91,4 +93,8 @@ type RcptOptions struct {
 	// Original recipient set by client.
 	OriginalRecipientType DSNAddressType
 	OriginalRecipient     string
+
+	// Time value of the RRVS= argument
+	// default time.Time value (Jan 1 1970) if not set by client
```

Comment:
"default time.Time value (Jan 1 1970) if not set by client" is confusing: the zero time is not in year 1970, it's in year 1.

I'd suggest replacing the sentence with "Left as the zero time if unset." (or dropping the sentence completely).